flybywiresim/a32nx
# feat(nd): VOR/ADF needles on ARC screen

Fixes #3055

To be sequenced after #3884 

## Summary of Changes

- Show VOR/ADF needles on ARC page
- Hide VOR/ADF needles on ILS page (I don't think they're supposed to be there looking at FCOM)
- Split control of needles so captain's are controlled by switches on captain's side and vice-versa.

## Screenshots (if necessary)


![image](https://user-images.githubusercontent.com/9995998/110580413-8388ee00-81cd-11eb-8f7e-2f42c0831a7a.png)


## References

![image](https://user-images.githubusercontent.com/9995998/110580956-846e4f80-81ce-11eb-922e-d785d31e9cfc.png)




## Additional context

I took the needle SVG path from Asobo's latest patch. If there are any budding SVG artists they could probably be improved.



Discord username (if different from GitHub):

## Testing instructions

Tune ADFs and/or VORs:
- Turn on needles
- Check that needles are displayed on all pages except LS and PLAN
- Check that needles point in direction of tuned navaids
- Check that switches on EFIS panel control ND needles on same side of the cockpit
- Check that ADF needles are green and VOR needles are white


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
